### DISCUSSION TITLE
Problems with simple configuration

### DISCUSSION THREAD

**Discussion by msolari1589** [serilog-contrib/serilog-sinks-grafana-loki#235] [Q&A]
Hello, i'm having a lot of trouble to start with this. I have a simple console logger and i want to send it to grafana but i cant.

This is my how i'm doing it:

I have a vm with windows, firewall is down.

[Main class]
- I try the long uri too and have the same result
![image](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/assets/60531077/9845a4d4-6b77-4f26-964a-ce598c5f66fa)

[Console output]
![image](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/assets/60531077/e1a45da7-38ff-49ad-b563-8d13ca037906)

[Loki config yaml]
![image](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/assets/60531077/14e950d9-2cd5-472f-9fff-48f2289212e2)

[Loki output]
![image](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/assets/60531077/bcbcc999-6a89-4d3a-a705-2dacfab7dd9e)

[Grafana datasource]
![image](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/assets/60531077/3cd0befd-9b35-4257-8090-43459b13d8e2)

[Grafana error]
![image](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/assets/60531077/4aadc20f-627f-45ef-933c-bafd0982df6c)

I dont know if i missing something here. Hope you can guide me with this.

Thanks for your help guys!

**Comment by mishamyte** [comment]
Hey @msolari1589!

Have you checked the [Common problems
](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Common-problems#sending-log-events-doesnt-seem-to-work) sections?

I would suggest enable SelfLogging (you could see how to do it in the sample app).
And also ensure that you are using the static logger (you configured), now the DI one.

Or configure logger into the DI if you inject `ILogger` into your worker
